06.bowling_object/shot.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class Shot
+  attr_reader :pin_count
+
+  def initialize(pin_count)
+    @pin_count = pin_count
+  end
+
+  def convert_to_int
```

Comment:
```suggestion
  def point
```

あえて動詞で始めず、プロパティ（属性）のように見せるメソッド定義にした方がコード上の馴染みがよくなりそうです